js/templates/search.es6.js

```diff
@@ -1,14 +1,11 @@
 const bel = require('bel');
 
 module.exports = function () {
-    return bel`<div class="js-search js-menu-section">
-        <form class="js-search-form" name="x">
-          <input type="text" autocomplete="off" placeholder="Search DuckDuckGo" name="q" class="js-search-input" value="${this.model.searchText}" />
```

Comment:
oh shoot -- sorry, i misunderstood you (its late here, my bad). you aren't lobbying for autocomplete "on", you're saying autocomplete "off" does have an effect. hadn't realized that it was a chrome thing, had been hunting around in our codebase for hints about what it does without realizing its a chrome setting. will fix tmro then.